tests/ProjectCodeTest.php

```diff
@@ -116,11 +116,33 @@ public function testHasThrowsAnnotation($method)
     public function testHasCorrespondingStaticAnalysisFile($method)
     {
         $doc = $method->getDocComment();
+        $methodName = $method->getName();
+
         if($doc === false || strpos($doc, '@psalm-assert') === false) {
             $this->addToAssertionCount(1);
             return;
         }
 
+        if (\in_array(
+            $methodName,
+            array(
+                'alnum',
+                'digits',
+                'upper',
+                'lower',
+                'readable',
```

Comment:
readable and writable don't have `@psalm-assert` on them right?
So they shouldn't be in this list.

Reply:
Correct, removing

Reply:
b5e64dcfc148d2c8d968865aee4deebccf2aee35